docs/Overview.md

```diff
@@ -223,32 +234,39 @@ The IS-05 schemas `receiver_transport_params_ndi.json` and `constraints_schema_r
 ```json
 [
   {
-    "interface_ip": "10.10.10.20",
-    "server_host": "10.10.10.10",
-    "server_port": 5960,
+    "machine_name": "ndi-machine-name",
     "source_name": "ndi-sender-unique-name",
-    "group_name": "camera1"
+    "source_url" : "...",
+    "source_address" : "10.10.10.123:5906"
   }
 ]
 ```
 
-**interface_ip**: 
-IP address of the network interface the receiver MUST use. If set to `auto` the receiver MUST determine which interface to use.
+machine name / source name SHOULD always be specified. Others MAY be specified.
+
+#### machine_name
+This indicates the device name of the Native NDI Sender that is to be connected, as utilized by the NDI SDK.
 
-**server_host**: 
-Hostname or IP hosting the NDI server. A `null` value indicates that the Receiver has not yet been configured.
+#### source_name
+The name of the Native NDI Sender stream in the NDI domain that is to be connected. This property MUST NOT be concatenatedwith `machine_name` in the format `machine_name(source_name)`.
 
-**server_port**: 
-Port for the NDI server. If set to `auto` the receiver MUST determine which port to use.
+> Informative notes: In the NDI domain, the `source_name` and `machine_name` are concatenatedin the format `machine_name(source_name)` when streams are discovered and connected, however in the `transport_params`, these properties are kept independent. 
 
-**source_name**: 
-The name of the NDI stream as declared by the NDI sender.
+#### source_url 
+The URL of the sender as utilized by the NDI SDK. The contents are proprietary to the NDI SDK and SHOULD NOT be interpreted. A controller SHOULD specify this parameter with values provided by a sender's `transport_params` or the NDI SDK discovery methods. If this parameter is set to `null`, the receiver MUST determine for itself the value of `source_url`.
 
-**group_name**: 
-The NDI group of the source, `null` indicates the default group.
+#### source_address
+The IP address and port that the sender is utilizing for the stream. If specified, it MUST be in the format `ip_address:port_number`, other wise it MUST be set to `null` if not known.
 
-Although the NDI Advanced SDK does allow NDI Native Devices to specify additional Receiver transport parameters, these parameters and properties SHOULD NOT be exposed in NMOS.
+## IS-05 Connection Bahavior 
+ Connections in the NDI domain can be specified in several different ways. In each case the Native NDI Receiver identifies the stream to be connected through at least one of:
+ - `machine_name` and `source_name`
+ - `source_url`
+ - `source_address` and `source_name`
+ - `source_address`
 
+ Native NDI Devices will not necessarily support all these methods, so Controllers and Devices SHOULD specify as much information as is known in `transport_params`. If the NDI Receiver is not provided with one or more of the above parameters, but is able to determine the additional parameters, it MUST report these updated `transport_params` upon activation.
```

Comment:
As said in the Slack channel I think this is asking too much to the Receiver ans I would remove this sentence "If the NDI Receiver is not provided with one or more of the above parameters, but is able to determine the additional parameters, it MUST report these updated `transport_params` upon activation."